wordpress-mobile/WordPress-Editor-iOS
# Fix media removal cancel upload

Fixes #627 

How to test:

Insert a media object on the post (video or image), check if on removal of the media event while the upload is happening a console message say "Media Removed: XXX"

Needs Review: @astralbodies
